ocaml/idl/datamodel_vm.ml

```diff
@@ -1701,6 +1701,31 @@ let restart_device_models =
     ~allowed_roles:(_R_VM_POWER_ADMIN ++ _R_CLIENT_CERT)
     ()
 
+let vm_uefi_mode =
+  Enum
+    ( "vm_uefi_mode"
+    , [
+        ( "user"
+        , "clears a VM's EFI variables related to Secure Boot and places it \
```

Comment:
This has label "user" but talks about setup and vice versa below. Is this correct?

Reply:
Indeed... Good catch thanks!